tursu/product_page/views.py

```diff
@@ -0,0 +1,30 @@
+from django.shortcuts import render
+from django.http import JsonResponse, HttpResponse
+from django.db.models import Q
+from product.models import Product
+
+def index(request):
+    try:
+        product_id = request.GET["id"]
+    except:
+        return JsonResponse([], safe=False)
```

Comment:
- In this case either an empty product dictionary or a http response with an error message would be a better response.